acme/jax/inference_server.py

```diff
@@ -45,12 +90,24 @@ class InferenceServerConfig:
 InferenceServerHandler = TypeVar('InferenceServerHandler')
 
 
+
+def reverso_oar_thing(oar_thing):
```

Comment:
Fine for now since it makes it work and we probably won't use this file anyways.